contributors/devel/api-conventions.md

```diff
@@ -1,3 +1,3 @@
 This file has moved to https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md.
 
-This file is a placeholder to preserve links.  Please remove by April 24, 2019 or the release of kubernetes 1.13, whichever comes first.
\ No newline at end of file
+This file is a placeholder to preserve links.  Please remove after 2019-07-01 or the release of kubernetes 1.15, whichever comes first.
```

Comment:
The date format is as per the style guide - https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md#moving-a-document.

(thanks for the doc again, @mrbobbytables! :heart: )